src/app/ideastable/ideas-table.component.html

```diff
@@ -0,0 +1,21 @@
+<div *ngIf="dataLength">
+  <table mat-table [dataSource]="dataSource" class="mat-elevation-z8">
+    <ng-container matColumnDef="name">
+      <th mat-header-cell *matHeaderCellDef>Name</th>
+      <td mat-cell *matCellDef="let idea"> {{idea.name}} </td>
+    </ng-container>
+    <ng-container matColumnDef="organization">
+      <th mat-header-cell *matHeaderCellDef>Organization</th>
+      <td mat-cell *matCellDef="let idea"> {{idea.organization}} </td>
+    </ng-container>
+    <ng-container matColumnDef="location">
+      <th mat-header-cell *matHeaderCellDef>Location</th>
+      <td mat-cell *matCellDef="let idea"> {{idea.location}} </td>
+    </ng-container>
+    <tr mat-header-row *matHeaderRowDef="columnNames"></tr>
+    <tr mat-row *matRowDef="let row; columns: columnNames"></tr>
+  </table>
+  <mat-paginator [length]="dataLength" [pageSize]="5" [pageSizeOptions]="[5, 10, 15]"
```

Comment:
maybe better to avoid magic numbers and move the pagination options to constants